src/server/pfs/server/obj_block_api_server.go

```diff
@@ -39,6 +41,36 @@ func newObjBlockAPIServer(dir string, objClient obj.Client) (*objBlockAPIServer,
 		dir:         dir,
 		localServer: localServer,
 		objClient:   objClient,
+		cache: groupcache.NewGroup("block", 1024*1024*1024*10,
```

Comment:
We should provide a way to enable/disable. How about (per your comments :-P) provide size as a config flag?